.storybook/polaris-readme-loader.js

```diff
@@ -187,52 +140,54 @@ function parseCodeExamples(data, file) {
     .split('###');
 
   const [, ...allExamples] = examplesAndHeader;
-  const filePath = file.split('polaris-react/').slice(-1)[0];
 
   if (allExamples.length === 0) {
-    throw new Error(
-      chalk`🚨 {red [${matter.data.name}]} No examples found in ${filePath}`,
+    console.log(
+      chalk`🚨 {red [${
+        matter.data.name
+      }]} No examples found. For troubleshooting advice see https://github.com/Shopify/polaris-react/blob/master/documentation/Component%20READMEs.md#troubleshooting'`,
     );
   }
 
-  const webExamples = allExamples.filter((example) =>
-    isExampleForPlatform(example, 'web'),
-  );
+  const nameRegex = /(.)*/;
+  const codeRegex = /```jsx(.|\n)*?```/g;
 
-  const examples = webExamples.map((example) => {
-    const nameMatches = example.match(/(.)*/);
-    const codeBlock = example.match(/```jsx(.|\n)*?```/g);
+  const examples = allExamples
+    .filter((example) => isExampleForPlatform(example, 'web'))
+    .map((example) => {
+      const nameMatches = example.match(nameRegex);
+      const codeBlock = example.match(codeRegex);
 
-    const name = nameMatches !== null ? nameMatches[0].trim() : '';
-    const code =
-      codeBlock !== null ? wrapExample(stripCodeBlock(codeBlock[0])) : '';
+      const name = nameMatches !== null ? nameMatches[0].trim() : '';
+      const code =
+        codeBlock !== null ? wrapExample(stripCodeBlock(codeBlock[0])) : '';
 
-    return {name, slug: slugify(name), code};
-  });
+      // TODO need to strip out android/ios examples
```

Comment:
Can't we use the same technique as before?

```js
  const webExamples = allExamples.filter((example) =>
    isExampleForPlatform(example, 'web'),
  );
```

Reply:
The filtering doesn't remove the android/ios content. `filterMarkdownForPlatform` does the job though